### ISSUE TITLE
tulip.timing() doesn't return all values

### ISSUE THREAD

**Issue by drepetto** [bwhitman/tulipcc#55]
The docs say:
# You can also change the timing and resolution on the fly. 
# This is helpful for getting higher FPS with lower resolution (less pixels)
(h_res, v_res, h_offscreen_px, v_offscreen_px, 
  hsync_back_porch, hsync_front_porch, hsync_pulse_width, 
  vsync_back_porch, vsync_front_porch, vsync_pulse_width) = tulip.timing() # returns current
tulip.timing(1024, 600, 256, 150, 139, 140, 20, 20, 12, 20) # sets, will erase all display RAM

but I get:
>>>tulip.timing()
>>>(1024, 600, 256, 150)

**Comment by bwhitman** [comment]
Yes, on desktop we should return -1 for the hardware display timings. Thank you, I can fix this soon
